R/mod_chart_example.R

```diff
@@ -12,6 +12,7 @@ mod_chart_example_ui <- function(id) {
     h2_tabstop("Second level"),
     nhs_card_tabstop(
       heading = "example chart title",
+      tabindex = 0,
```

Comment:
Need to specify this, as `nhs_card_tabstop` is expecting the content to be everything after the 2 named args.